test-framework/test-suites/integration/tests/add/test_add_host_interface.py

```diff
@@ -242,4 +242,69 @@ def test_add_host_interface_ip_auto(self, host):
 				'options': None,
 				'vlan': None
 			}
-		]
\ No newline at end of file
+		]
+
+
+	def test_add_host_interface_duplicate_default_interface(self, host):
+		# Add a dummy host
+		result = host.run('stack add host backend-8-8')
```

Comment:
Could you use a test fixture here to add the host + initial interface?  You might have to change the hostname.

Reply:
Stefan - A little more guidance: The test class already adds the backend-0-0 host via the class decorator: `@pytest.mark.usefixtures("add_host")`

So, you can just add the eth0 and eth1 interfaces to that host:
```
result = host.run('stack add host interface backend-0-0 interface=eth0 default=true')
assert result.rc == 0

... (checks and stuff)

result = host.run('stack add host interface backend-0-0 interface=eth1 default=true')
assert result.rc == 0
```